Makefile

```diff
@@ -131,13 +131,16 @@ tools/bin/controller-gen: go.mod
 	go build -o $@ sigs.k8s.io/controller-tools/cmd/controller-gen
 
 .PHONY: manifests
+
 manifests: tools/bin/controller-gen ## Generate manifests e.g. CRD, RBAC etc.
 	cd apis; ../$< $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=../config/crd/bases
+	$< rbac:roleName=manager-role paths="./..." output:rbac:artifacts:config=config/rbac
 	$(KUSTOMIZE) build config/default > config/render/capm3.yaml
 
 .PHONY: generate
 generate: tools/bin/controller-gen ## Generate code
 	cd apis; ../$< object:headerFile="../hack/boilerplate.go.txt" paths="./..."
+	$< object:headerFile="hack/boilerplate.go.txt" paths="./..."
```

Comment:
Out of curiousity, do we need both generate and manifests, or might just manifests suffice (at least for now)?

Reply:
this is to just make sure that if we add something in another dir, it's not going confuse the hell out of someone for hours.